src/OrleansAWSUtils/Reminders/DynamoDBReminderTable.cs

```diff
@@ -157,6 +183,13 @@ private static ReminderEntry Resolve(Dictionary<string, AttributeValue> item)
             };
         }
 
+        /// <summary>
+        /// Remove one row from the reminder table 
+        /// </summary>
+        /// <param name="grainRef"> specific grain ref to locate the row </param>
+        /// <param name="reminderName"> remider name to locate the row </param>
+        /// <param name="eTag"> e tag </param>
+        /// <returns> A Task object to indicate the result of this aync method </returns>
```

Comment:
I'd say `Return true if a row was removed'. The fact that it's a task is not the important part of the return value.